### DISCUSSION TITLE
[Question] Public demo

### DISCUSSION THREAD

**Discussion by lonix1** [elsa-workflows/elsa-core#597] [Q&A] [answered]
Is there a public demo somewhere, so we can play around with it without setting it up first?

**Comment by steve99** [comment, marked as answer]
Don't believe so, but its really easy to take an demo and build it, to see the basics, or see the docs which have screenshots or search online for other sites showing examples.

**Reply by sfmskywalker** [reply]
Indeed, there's no public demo at the moment, but I have it on the back of my worklog.
